server/src/main/java/org/opensearch/wlm/stats/QueryGroupStats.java

```diff
@@ -37,19 +39,26 @@
  *     ...
  * }
  */
-public class QueryGroupStats implements ToXContentObject, Writeable {
+public class QueryGroupStats extends BaseNodeResponse implements ToXContentObject, Writeable {
```

Comment:
I am wondering if we should rename this class also to `WlmStats` since, we might have more other stats in addition to `QueryGroupStats` within this class